### PR TITLE
Remove cassandra chart

### PR DESCRIPTION
**Pull request by kragniz**
We should encourage users to directly use the CassandraCluster and ElasticsearchCluster resources. The helm chart creates a layer of abstraction I don't think we need, and there is no corresponding elasticsearch chart.

This instead uses the same templating elasticsearch uses for e2e testing (envsubst).

/cc @mattbates 

```release-note
Remove cassandra chart
```

### CODE REVIEW

File: hack/testdata/cass-cluster-test.template.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+apiVersion: "navigator.jetstack.io/v1alpha1"
+kind: "CassandraCluster"
+metadata:
+  name: "test"
```
Comment (wallrj) [inline hack/testdata/cass-cluster-test.template.yaml]: This should be a CASS_NAME parameter I think

Comment (kragniz) [reply hack/testdata/cass-cluster-test.template.yaml]: :+1:

**Comment by kragniz** [comment]
Now rebased on master with tests passing again

**Pull request merged**